website/templates/public/pages/getting_started.mako

```diff
@@ -65,7 +65,7 @@
         </div>
     <div class="col-sm-9">
 
-        <div  id="start"  class="col-sm-12">
+        <div  id="start"  class="col-sm-12" style="margin-left: 40px">
```

Comment:
In general, we want to avoid these inline styles. Is there a better way to achieve using Bootstrap's built-in classes?